src/main/java/core/org/akaza/openclinica/web/table/sdv/SDVUtil.java

```diff
@@ -959,7 +967,22 @@ public void formatColumns(HtmlTable table, String[] columnNames, HttpServletRequ
                 }
                 crfStatusBuilder.append("<center><a title='" + statusTitle + "' alt='" + statusTitle + "' class='" + statusIconClassName + "' accessCheck' border='0'/></center>");
                 tempSDVBean.setCrfStatus(crfStatusBuilder.toString());
-            tempSDVBean.setSubjectEventStatus("<center><a title='"+eventCrf.getStudyEvent().getWorkflowStatus()+"' alt='"+eventCrf.getStudyEvent().getWorkflowStatus()+"' class='"+STUDY_EVENT_WORKFLOW_ICONS.get(eventCrf.getStudyEvent().getWorkflowStatus())+"' accessCheck' border='0'/></center>");
+                String studyEventStatusName ;
+                String studyEventStatusDisplayName;
```

Comment:
I wad not able to set the STUDY_EVENT_WORKFLOW_ICONS map with getDisplayValue() as key, Since it is causing bean creation error as it is static. Hence i use toString() as key and used to varaiables, one for displaying and other for fetching from map.